rpc_sidecar/src/rpcs/info.rs

```diff
@@ -540,6 +538,36 @@ mod tests {
 
     use super::*;
 
+    #[tokio::test]
+    async fn get_deploy_result_none_execution_info_should_serialize_to_null() {
+        let rng = &mut TestRng::new();
+        let deploy = Deploy::random(rng);
+        let result = GetDeployResult {
+            api_version: CURRENT_API_VERSION,
+            deploy,
+            execution_info: None,
+        };
+
+        let json_value = serde_json::to_value(&result).unwrap();
+
+        assert!(json_value.get("execution_info").unwrap().is_null());
```

Comment:
```suggestion
        assert!(json_value.get("execution_info").expect("should have execution_info").is_null());
```
nitpick, but we usually use `expect` to make test errors more readable

Reply:
👍